src/fetch-api/ExtensionInstallations.js

```diff
@@ -0,0 +1,59 @@
+import _ from 'lodash';
+import { fetchResource } from '../resources/Resource';
+import extensionInstallationResource, { INSTALLATIONS } from '../resources/extensionInstallations';
+
+export default class ExtensionInstallations {
+  constructor(config) {
+    this.config = config;
+
+    this.get = this.get.bind(this);
+    this.update = this.update.bind(this);
+    this.getSettings = this.getSettings.bind(this);
+    this.updateSettings = this.updateSettings.bind(this);
+
+    this.resource = extensionInstallationResource(config);
+  }
+
+  get(config = {}) {
+    const resolvedConfig = { ...this.config, ...config };
+    const { extensionInstallationId } = resolvedConfig;
```

Comment:
This will also cause problems when we start having multiple extensions in the same environment, for example when settings pages are running directly in the builder. Can you change this so that the extension id must always be passed in as an argument, and expose this id to the settings page trough props?

The same applies for shortcuts, screens, etc.